server/run.sh

```diff
@@ -10,6 +10,8 @@ do
       python3 -m pip install -r requirements.txt;;
     "--lint" )
       flake8 tests halomod_app;;
+    "--fix" )
```

Comment:
This looks like it is a lint fix. `fix` seems very vague and Steven / future maintainers might not know what is meant here. Maybe `--lint-fix`?

Reply:
I wanted it to look like sh run.sh --lint --fix (kinda like the npm one). What do you think?

Reply:
Oh that definitely makes sense. Can that be put in the readme?

Reply:
Never-mind. Saw the most recent change for this 😄